python/dgl/distributed/dist_context.py

```diff
@@ -41,6 +45,126 @@ def _init_rpc(ip_config, num_servers, max_queue_size, net_type, role, num_thread
         traceback.print_exc()
         raise e
 
+
+class MpCommand(Enum):
+    """Enum class for multiprocessing command"""
+    INIT_RPC = 0  # Not used in the task queue
+    SET_COLLATE_FN = 1
+    CALL_BARRIER = 2
+    DELETE_COLLATE_FN = 3
+    CALL_COLLATE_FN = 4
+    CALL_FN_ALL_WORKERS = 5
+    FINALIZE_POOL = 6
+
+
+def init_process(_, rpc_config, mp_contexts):
```

Comment:
why do you want to have the first argument while not using it?

Reply:
Removed